setup.py

```diff
@@ -567,7 +622,7 @@ def make_relative_rpath(path):
                            THC_LIB,
                            THCUNN_LIB,
                            make_relative_rpath('../lib'),
-                       ]
+                       ] + [_C_LIB] if _C_LIB is not None else []
```

Comment:
This is needed because there's no RTLD_GLOBAL for Windows, right?

Reply:
Yes, that's it.